pkg/sql/logictest/testdata/logic_test/information_schema

```diff
@@ -2949,20 +2952,20 @@ SELECT table_catalog, table_schema, table_name, column_name, column_comment
 FROM information_schema.columns
 WHERE table_name = 't1'
 ----
-table_catalog  table_schema  table_name  column_name    column_comment
-test           public        t1          id             'identification'
-test           public        t1          name           'character'
-test           public        t1          class          ·
-test           public        t1          rowid          ·
+table_catalog  table_schema  table_name  column_name  column_comment
+test           public        t1          id           identification
+test           public        t1          name         character
+test           public        t1          class        NULL
+test           public        t1          rowid        NULL
```

Comment:
Do we want NULLs here, or should those be replaced by empty strings? I honestly can't remember exactly. Feel free to disregard if this doesn't matter.